recipes/pybind11_json/all/test_package/CMakeLists.txt

```
@@ -0,0 +1,11 @@
+cmake_minimum_required(VERSION 3.1)
+project(test_package)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup(TARGETS)
+
+find_package(pybind11_json REQUIRED CONFIG)
+
+add_executable(${PROJECT_NAME} test_package.cpp)
+target_link_libraries(${PROJECT_NAME} pybind11_json::pybind11_json)
```

Looks like the official target is without namespace: https://github.com/pybind/pybind11_json/blob/0fbbe3bbb27bd07a5ec7d71cbb1f17eaf4d37702/CMakeLists.txt#L133-L135
```suggestion
target_link_libraries(${PROJECT_NAME} pybind11_json::pybind11_json)  # FIXME: target should be pybind11_json
```

---

Good point. How to fix this? When I remove the namespace, the build fails. Should I propose to add NAMESPACE in https://github.com/pybind/pybind11_json/blob/0fbbe3bbb27bd07a5ec7d71cbb1f17eaf4d37702/CMakeLists.txt#L133-L135?